src/addWords/AddWordsPage.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import ListPageBase from '../common/ListPageBase';
+import AddWordsListItem from './AddWordsListItem';
+import Button from '@material-ui/core/Button';
+import './AddWordsPage.css';
+
+const BASE_NUM_ROWS = 10;
+
+class AddWordsPage extends ListPageBase {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ...this.state,
+      loading: false,
+      listItemTag: AddWordsListItem,
+      items: [...this.state.items, ...this.getEmptyItems_()],
+      disabled: true,
+    };
+  }
+
+  getEmptyItems_() {
+    const items = [];
+
+    for (let i = 0; i < BASE_NUM_ROWS; i++) {
+      items.push({
+        id: this.state.items.length + i,
+        english_word: '',
+        sound_link: '',
+        translation: '',
+        transliteration: '',
+      });
+    }
+
+    return items;
+  }
+
+  addRows_() {
+    this.setState({
+      items: [...this.state.items, ...this.getEmptyItems_()]
+    });
+  }
+
+  renderBulkUpload() {
```

Comment:
Maybe this should be "private"? renderBulkUpload_

Reply:
fixed.